types.go

```diff
@@ -185,6 +185,15 @@ type Lens struct {
 		KeyFile  string `json:"key_file"`
 	} `json:"tls"`
 	AuthKey string `json:"auth_key"`
+	Options struct {
+		Queue struct {
```

Comment:
`Queue` is technically a set of sub-options of `Engine`

Reply:
Ah true